exhibitor-core/src/main/java/com/netflix/exhibitor/core/azure/AzureClientImpl.java

```diff
@@ -0,0 +1,63 @@
+package com.netflix.exhibitor.core.azure;
+
+import com.microsoft.azure.storage.CloudStorageAccount;
+import com.microsoft.azure.storage.blob.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+
+public class AzureClientImpl implements AzureClient {
+    private final Logger log = LoggerFactory.getLogger(getClass());
+    private final String storageConnectionString;
+
+    public AzureClientImpl(AzureCredential credentials) {
+        this.storageConnectionString =
+            "DefaultEndpointsProtocol=http" +
+            ";AccountName=" + credentials.getAccountName() +
+            ";AccountKey=" + credentials.getAccountKey();
+    }
+
+    @Override
+    public CloudBlobClient getClient() throws Exception {
+        CloudStorageAccount storageAccount = CloudStorageAccount.parse(this.storageConnectionString);
+        return storageAccount.createCloudBlobClient();
+    }
+
+    @Override
+    public CloudBlob getBlob(String containerName, String uri) throws Exception {
+        CloudBlobContainer container = getClient().getContainerReference(containerName);
+        return container.getBlockBlobReference(uri);
```

Comment:
Azure documentation was initially unclear, but it's not actually a URI - renaming to `blobName`